src/components/Capture/capture.js

```diff
@@ -97,11 +98,15 @@ class Capture extends Component {
       return
     }
     payload = {...payload, documentType}
-    if (this.props.side === 'back' && !this.props.useWebcam) {
-      payload = {...payload, valid: true}
+    if (this.props.useWebcam) {
+      postToServer(this.createJSONPayload(payload), serverUrl, token,
+        (response) => this.onServerResponse(response),
```

Comment:
the `onServerResponse` response inside is looking at the `valid` property. is this something the api is returning? Also, we should store the document/face id the api gives us and return that to the user instead.